pkgs/tools/networking/dae/default.nix

```diff
@@ -37,6 +41,20 @@ buildGoModule rec {
   # network required
   doCheck = false;
 
+  assetsDrv = symlinkJoin {
+    name = "dae-assets";
+    paths = [ v2ray-geoip v2ray-domain-list-community ];
+  };
+
+  postInstall = ''
+    install -Dm555 "$GOPATH"/bin/dae $out/bin/dae
```

Comment:
Isn't this line redundant? (the binary should already be installed in the installPhase)